source/jobs/pre-requisitos.md

```diff
@@ -7,7 +7,7 @@ extends: _layouts.job
 * Conhecer as leis que regem o nosso ramo do cooperativismo:
   * [LEI Nº 5.764, DE 16 DE DEZEMBRO DE 1971](http://www.planalto.gov.br/ccivil_03/Leis/L5764.htm)
   * [LEI Nº 12.690, DE 19 DE JULHO DE 2012](http://www.planalto.gov.br/CCIVIL_03/_Ato2011-2014/2012/Lei/L12690.htm)
-* Conhecer e aceitar os termos de nosso [Estatuto](https://gitlab.com/librecodecoop/estatuto) e [Regimento Interno](https://gitlab.com/librecodecoop/regimento-interno)
+* Conhecer e aceitar os termos de nosso [Estatuto](https://gitlab.com/librecodecoop/estatuto/-/jobs/6587309602/artifacts/file/estatuto.pdf) e [Regimento Interno](https://gitlab.com/librecodecoop/regimento-interno/-/jobs/3543571376/artifacts/file/regimento.pdf)
```

Comment:
This link will go to a specific build. This isn't good because the build will change after next merge.

The previous link go to repository. Maybe we can highlight better the link to PDF at README of these repository.